src/main/java/com/github/piedpiper/node/stepfunctions/StepFunctionsBaseHandler.java

```diff
@@ -0,0 +1,35 @@
+package com.github.piedpiper.node.stepfunctions;
+
+import com.amazonaws.services.stepfunctions.AWSStepFunctions;
+import com.amazonaws.services.stepfunctions.AWSStepFunctionsClientBuilder;
+import com.github.piedpiper.aws.AccessKeySecretKeyCredentialProvider;
+import com.github.piedpiper.node.NodeInput;
+import com.github.piedpiper.node.NodeOutput;
+
+public abstract class StepFunctionsBaseHandler extends StepFunctionsBaseNode {
```

Comment:
Bad idea to make it abstract. Composition is always better than inheritaqnce. And in your case you can get step function handler using a factory for each method. Refer to RESTServiceNode